torchmultimodal/models/flava/text_encoder.py

```diff
@@ -200,19 +201,18 @@ def flava_text_encoder(
         type_vocab_size=type_vocab_size,
         max_position_embeddings=max_position_embeddings,
         layer_norm_eps=layer_norm_eps,
-        hidden_dropout_prob=hidden_dropout_prob,
+        hidden_dropout_prob=dropout,
     )
 
-    encoder = FLAVATransformerEncoder(
-        hidden_size=hidden_size,
-        num_attention_heads=num_attention_heads,
-        num_hidden_layers=num_hidden_layers,
-        hidden_dropout_prob=hidden_dropout_prob,
-        intermediate_size=intermediate_size,
-        intermediate_activation=intermediate_activation,
-        attention_probs_dropout_prob=attention_probs_dropout_prob,
+    encoder = TransformerEncoder(
+        n_layer=num_hidden_layers,
+        d_model=hidden_size,
+        n_head=num_attention_heads,
+        dim_feedforward=intermediate_size,
+        activation=intermediate_activation,
         layer_norm_eps=layer_norm_eps,
```

Comment:
where was pad_token_id used, now that we are removing it?

Reply:
it was not used anywhere in `FLAVATransformerEncoder` or `FLAVATransformerLayer`